smarts/core/tests/test_renderers.py

```diff
@@ -104,6 +105,7 @@ def test_renderer(self):
 
 
 def test_multiple_renderers(scenario):
+    assert p3dThread.isThreadingSupported()
```

Comment:
This was mentioned by @sah-huawei as something we should check. At minimum it should help narrow down the cause of the test failure.

Reply:
Darn.  I was hoping it would be as simple as that, but it looks like not (given that the tests for this commit pass).

I would still opt for leaving this assert in (i.e., I think we can go ahead and merge this PR even though it didn't find the problem in Issue #1175) because it could be useful for other users who've already installed panda3d on their own system.